inst/rmarkdown/templates/jds/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,271 @@
+survival analysis and reliability engineering, it has a vast domain of other
+applications.
+
+Equations are always parts of sentences, so they need to have
+appropriate punctuations. To evaluate the distribution of a normal
+variable, one use
+\begin{equation}
+  \label{eq:cdf}
+  \Pr(Z \le t) = \Phi\left(\frac{Z - \mu}{\sigma} \right),
+\end{equation}
+where $Z$ follows a $N(\mu, \sigma^2)$ distribution.
+Equations can be referenced by \texttt{$\backslash$eqref}.
+When $\mu = 0$ and $\sigma = 1$, the $Z$ in Equation~\eqref{eq:cdf}
+becomes a standard normal variable.
+
+
+Multiline equations can be presented with the \texttt{align}
+environment. For example,
+\begin{align*}
+  g_{\mu}(\phi) = 0, \\
+  g_{\mu}(X) = 1.
+\end{align*}
+
+
+An equation that is not referenced should not be labeled. The starred
+version of the \texttt{equation} and \texttt{align} are for this purpose.
+
+# Tables {#sec:tabs}
+
+We recommend \LaTeX\ package `booktabs` for professional
+looking tables. Its toprule and bottomrule are thicker than midrule.
+
+A professional table contains no vertical lines.
+
+Table \ref{tab:realdata} is an illustration.
+
+\begin{table}[tbp]
+  \caption{Analysis results for real data. Point
+    estimates (EST) from both two-piece method and marginal method are
+    reported. Standard error of point estimates are evaluated by
+    parametric bootstrap (SE).}%
+  \label{tab:realdata}
+  \centering
+  \begin{tabular}{llrrrr}
+    \toprule
+    \multicolumn{1}{l}{Season} & \multicolumn{1}{l}{Parameter}
+                               & \multicolumn{2}{c}{Two-piece method} & \multicolumn{2}{c}{Marginal method}                         \\
+    \cmidrule(r){3-4} \cmidrule(r){5-6}
+                               &                                      & EST                                 & SE    & EST   & SE    \\
+    \midrule
+    Summer                     & $\lambda_1$                          & 2.841                               & 0.459 & 1.090 & 0.280 \\
+                               & $\lambda_0$                          & 0.179                               & 0.014 & 0.158 & 0.015 \\
+                               & $\sigma$                             & 1.335                               & 0.106 & 0.999 & 0.104 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 1.854                               & 0.087 & 1.879 & 0.078 \\
+    Winter                     & $\lambda_1$                          & 6.225                               & 0.825 & 4.720 & 0.711 \\
+                               & $\lambda_0$                          & 0.118                               & 0.010 & 0.114 & 0.009 \\
+                               & $\sigma$                             & 1.506                               & 0.095 & 1.454 & 0.089 \\
+                               & $\sigma_\epsilon (\times 10^{-2})$   & 0.908                               & 0.036 & 0.934 & 0.043 \\
+    \bottomrule
+  \end{tabular}
+\end{table}
+
+
```

Comment:
Should we show example using **knitr** with `booktabs = TRUE` ? 

This would allow also the test of this format to check that knitr tables are working